src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1005,6 +1003,8 @@ public void cannotSyncPostWithDuplicateId() {
   public Response postSynchronousBatchWithExistingId(String subPath) {
     JsonArray itemsArray1 = threeItems();
     JsonArray itemsArray2 = threeItems();
+    itemsArray2.getJsonObject(0).put("barcode", "new0");
+    itemsArray2.getJsonObject(2).put("barcode", "new2");
```

Comment:
Are these details important to the scenario this test is intending to cover? 

If not, could barcode generation be included in the `threeItems` method instead of the main body of the test.

Reply:
threeItems() has always returned the same barcodes, now it returns random barcodes to prevent a duplicate barcode error.